data/postsrsd.conf.in

```diff
@@ -152,22 +156,30 @@ hash-minimum = 4
 always-rewrite = off
 
 # Execute PostSRSd as unprivileged user
-# If you set this highly recommended option, PostSRSd will drop root
-# privileges and switch to the configured user before it enters the main loop
-# to handle untrusted input.
+# Drop root privileges and run as this user before entering the main loop and
+# handling untrusted input. To prevent PostSRSd from changing users, set this to
+# the empty string.
 #
-# Example:
+# Default if unset:
```

Comment:
I don't like the "Default if unset" extra paragraphs. If a default is important to document, I'd rather have it as part of the description text. Also, I have intentionally and explicitly set all values to their default values in the example config, which serves the same purpose.

Reply:
Done.